aw/Helpers/AWApiHelper.swift

```diff
@@ -183,6 +190,7 @@ struct AWApiHelper {
             }
             DefaultsManager.lastUpdated = Date()
             DefaultsManager.favoritesLastUpdated = Date()
+            DefaultsManager.fetchingreaches = false
```

Comment:
If an error is thrown, is this statement guaranteed to be called?

Reply:
No, but I've added the same line to TabViewController so that it will be called if someone re-launches the app which should make sure it gets cleared. 

There are enough nested async scopes that I don't think `defer {` will do the trick

Reply:
Ok, keep this code in mind if we have any app loading issues. Code like this, async with local state, can be extremely difficult to debug.